src/PhpDoc/TypeNodeResolver.php

```diff
@@ -242,6 +243,12 @@ private function resolveIdentifierTypeNode(IdentifierTypeNode $typeNode, NameSco
 					new AccessoryNonEmptyStringType(),
 				]);
 
+			case 'non-falsy-string':
```

Comment:
just noting here so it will not be forgotten:
on [twitter there was a discussion going](https://twitter.com/seldaek/status/1552583227893743616) on whether this type should better be named `truthy-string` instead (and keep `'non-falsy-string'` as a deprecated alias)